frontend/src/citizen-frontend/personal-details/PersonalDetails.tsx

```diff
@@ -117,10 +117,13 @@ export default React.memo(function PersonalDetails() {
 
             return (
               <>
-                {email === null && (
+                {(!email || !phone) && (
```

Comment:
Now both of the warnings are shown even if one of them is already filled.

Reply:
The alert is shown if either email or phone is missing, and the alert text reflects dynamically what is missing?

Reply:
🤦 got it, did two refactoring that cancelled each other out